### PR TITLE
Update RELEASED_VERSIONS

### PR DESCRIPTION
**Pull request by roxbot**
This is an automated PR created from https://github.com/stackrox/stackrox/pull/2004.
  Add entry into the RELEASED_VERSIONS file

### CODE REVIEW

File: RELEASED_VERSIONS [hunk 1]
```diff
@@ -16,3 +16,4 @@
 3.7.0 3.69.0  # Rox release 3.69.0 by 0x656b694d at Tue Mar 15 17:01:36 UTC 2022
 3.7.3 3.69.1  # Rox release 3.69.1 by 0x656b694d at Wed Apr  6 21:13:36 UTC 2022
 3.8.1 3.70.0  # Rox release 3.70.0 by JoukoVirtanen at Fri May 27 21:22:32 UTC 2022
+3.8.1 3.68.2  # Rox release 3.68.2 by JoukoVirtanen at Fri Jun 17 01:22:13 UTC 2022
```
Comment (Molter73) [inline RELEASED_VERSIONS:19]: Shouldn't `3.68.2` be tied to collector's `3.6.2`? Whatever happened here?

Comment (erthalion) [reply RELEASED_VERSIONS:19]: Yep, looks weird. Maybe @JoukoVirtanen know more?

Comment (JoukoVirtanen) [reply RELEASED_VERSIONS:19]: Good catch. The script that creates this PR is usually run in CircleCI, but this time it was created by running a script manually. Something may have gone wrong with that.

Comment (JoukoVirtanen) [reply RELEASED_VERSIONS:19]: 3.68.2 is actually tied to collector 3.6.4.

**Review by JoukoVirtanen — APPROVED** [review]

**Pull request merged**